firmware/lib/TestStandCommController/src/TestStandCommController.cxx

```diff
@@ -0,0 +1,36 @@
+#include "Arduino.h"
+#include "TestStandCommController.h"
+
+static uint8_t send_buf[MSG_DATA_LENGTH_MAX];
+
+TestStandCommController::TestStandCommController(SerialDevice& device) : TestStandComm(device)
+{
+    // Nothing else to do
+}
+
+bool TestStandCommController::log(LogLevel log_level, const char *fmt, ...)
```

Comment:
What does `*fmt` stand for? What is `...`

Reply:
fmt is short for format, I think it's a pretty standard abbreviation I've seen in C code

... is varargs (variable arguments). It basically lets you pass any number of arguments you want to the function. This is how printf works (where you pass a format, followed by as many variables as you want).